### PR TITLE
remove moment.js dependency

### LINKED ISSUE #278: Replace cljsjs.moment with goog.date helpers
### User Story

As a user, I want to OpenBounty to load and run fast so that I can get things done.

### Description

*Type*: TechDebt

*Summary*: Currently we depend on `cljsjs.moment` for a single function:

```clj
;; commiteth/common.cljs
(defn moment-timestamp [time]
  (let [now (.now js/Date.)
        js-time (clj->js time)]
    (.to (js/moment.utc) js-time)))
```

Moment.js adds ~20k (gzipped) to the build which impacts download and JS parsing time. Using `goog.date` we are able to achieve the same while also benefitting from advanced compilation strategies such as dead code elimination. I expect that with `goog.date` we might end up with this function not taking more than 1kb. 

[goog.i18n.DateTimeFormat](https://google.github.io/closure-library/api/goog.i18n.DateTimeFormat.html) might be all that's needed here.

### PR DESCRIPTION
**Pull request by rcullito**
and replace with goog.date.relative.

Fixes #278 

One aspect of this that I was unable to discern locally was what type the [time] parameter was previously? I assumed milliseconds, but am happy to add additional logic to the extent that it is stored in a different format.

### CODE REVIEW

File: src/cljs/commiteth/common.cljs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   (:require [reagent.core :as r]
             [re-frame.core :as rf]
             [clojure.string :as str]
-            [cljsjs.moment]))
+            [goog.date.relative :as relative]))
```
Comment (martinklepsch) [inline src/cljs/commiteth/common.cljs]: When using a namespace only a few times I often prefer to not alias it, i.e.
```
[goog.date.relative]
```
This makes it easier to read the code without keeping all the aliases in your head.

This is general feedback, you don't need to change this to get this PR merged. 👍

Comment (rcullito) [reply src/cljs/commiteth/common.cljs]: that feedback makes sense! just updated without the alias for the sake of consistency

**Comment by rcullito** [comment]
Thank you for your feedback @martinklepsch! I love the project though did find some of the setup steps a bit tricky. 

Mainly I was flipping between the README and the riot chat for getting the geth node working (and am still not wholly confident in my process). Do you think this is strictly necessary to have in order to `lein run` or would there be value in designing a mock service so that this was not required to get the webapp itself working? 

I also wasn't wholly sure what role the github and Oauth apps are playing in this repo working locally, so more of an explanation around that might be helpful in order to come up with the right credentials? Thanks for bearing with all of my questions as I get up to speed!  Excited to keep contributing. 😺

File: src/cljs/commiteth/common.cljs [hunk 2]
```diff
@@ -23,10 +23,10 @@
                             {:value item}
                             item]))]))
 
-(defn moment-timestamp [time]
-  (let [now (.now js/Date.)
-        js-time (clj->js time)]
-    (.to (js/moment.utc) js-time)))
+(defn relative-time [time]
```
Comment (pablodip) [inline src/cljs/commiteth/common.cljs]: How about something like `time-millis->relative-str`?

**Comment by pablodip** [comment]
@rcullito Could you please describe steps for QA to verify that sessions are working properly? You can do it in the description of the pull request.

**Comment by martinklepsch** [comment]
@rcullito thanks for taking the time to write up your observations about the setup process:

> Mainly I was flipping between the README and the riot chat for getting the geth node working (and am still not wholly confident in my process). 

We're currently investigating using Infura to simplify this process. A mock service may also be an option but I believe Infura might be easier/quicker to implement for now.

> I also wasn't wholly sure what role the github and Oauth apps are playing in this repo working locally, so more of an explanation around that might be helpful in order to come up with the right credentials? 

Yeah that makes sense. It's a bit of a pain to set up all that stuff. I think for this specifically having some development authentication service which does not rely on Github could be very useful. I'll note that and see if we can make it happen.

> Thanks for bearing with all of my questions as I get up to speed! Excited to keep contributing. 😺

Awesome to have you on board! 🤗 

---


> @rcullito Could you please describe steps for QA to verify that sessions are working properly? You can do it in the description of the pull request.

@pablodip this PR has nothing to with sessions maybe you wanted to add that to #288? 

--- 

In the spirit of [optimistic merging](http://hintjens.com/blog:106) I will merge this PR. I didn't test it locally but according to the Moment.js / Closure docs the changes seem sound and my impression is that @rcullito did test this properly. If it's not working, I'll take the blame.

**Comment by pablodip** [comment]
> @pablodip this PR has nothing to with sessions maybe you wanted to add that to #288?

Sorry, I forgot to modify the message, but was intended for this PR.

> In the spirit of optimistic merging I will merge this PR. I didn't test it locally but according to the Moment.js / Closure docs the changes seem sound and my impression is that @rcullito did test this properly. If it's not working, I'll take the blame.

Let's try to pass all PRs that modify code that is used by the system (not lein config for example) pass through QA, please, especially because we almost don't have any automated tests and don't use CI to validate.

**Comment by churik** [comment]
Tested in #311

**Pull request merged**